Tools/CISupport/build-webkit-org/steps.py

```diff
@@ -1262,7 +1262,7 @@ class PrintConfiguration(steps.ShellSequence):
     warnOnFailure = False
     logEnviron = False
     command_list_generic = [['hostname']]
-    command_list_apple = [['df', '-hl'], ['date'], ['sw_vers'], ['system_profiler', 'SPSoftwareDataType', 'SPHardwareDataType'], ['xcodebuild', '-sdk', '-version']]
+    command_list_apple = [['df', '-hl'], ['date'], ['sw_vers'], ['system_profiler', 'SPSoftwareDataType', 'echo TimezoneVers: $(cat /usr/share/zoneinfo/+VERSION)', 'SPHardwareDataType'], ['xcodebuild', '-sdk', '-version']]
```

Comment:
The is a list of commands, each sub-list is the command argument. What you're trying to do is basically add `['echo', 'TimezoneVers: $(cat /usr/share/zoneinfo/+VERSION)']`. We need to test this on the staging instance, though, because `$(cat ...)` bit indicates you're trying to use shell scripting, so you probably need to do `['sh', 'echo TimezoneVers: $(cat /usr/share/zoneinfo/+VERSION)']` so that you are invoke with the shell instead of just subprocess.